inference-engine/tests/functional/plugin/cpu/shared_tests_instances/behavior/layout.cpp

```diff
@@ -27,10 +29,9 @@ namespace {
     INSTANTIATE_TEST_CASE_P(smoke_BehaviorTests, LayoutTest,
             ::testing::Combine(
                     ::testing::Values(InferenceEngine::Precision::FP32),
-                    ::testing::Values(CommonTestUtils::DEVICE_CPU),
+                    ::testing::Values(CommonTestUtils::DEVICE_CPU, "HETERO:CPU"),
```

Comment:
let's also enable for `HETERO:TEMPLATE` and `MULTI_TEMPLATE` [here](https://github.com/openvinotoolkit/openvino/blob/master/docs/template_plugin/tests/functional/shared_tests_instances/behavior/layout.cpp)